test_scripts/API/VehicleData/StabilityControlsStatus/011_GetVehicleData_StabilityControlsStatusInvalid_WithOtherValidVD.lua

```diff
@@ -0,0 +1,91 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving stabilityControlsStatus (with invalid parameters)
+-- and some other vehicle data (with valid parameters) via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+--
+-- Steps:
+-- 1) App sends GetVehicleData (with stabilityControlsStatus = true) request to SDL
+-- SDL does:
+--  - send VehicleInfo.GetVehicleData (with stabilityControlsStatus = true, speed = true) request to HMI
+-- HMI sends VehicleInfo.GetVehicleData 
+--  - response "SUCCESS", stabilityControlsStatus with invalid parameters
+-- SDL does:
+--  - send to app GetVehicleData response with (success: false, resultCode: "GENERIC_ERROR")
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Local Variables ]]
+local reqParams = { 
+  stabilityControlsStatus = true,
+  speed = true
+}
+
+-- trailerSwayControl is updated to boolean instead of VehicleDataStatus type
+local hmiResParams_invalidTrailerSwayControl = {
+  stabilityControlsStatus = {
+    escSystem = "ON",
+    trailerSwayControl = true
+  },
+  speed = 30.2
+}
+
+-- escSystem is updated to integer instead of VehicleDataStatus type
+local hmiResParams_invalidEscSystem = {
+  stabilityControlsStatus = {
+    escSystem = 20,
+    trailerSwayControl = "OFF"
+  },
+  speed = 30.2
+}
+
+-- escSystem is updated to float instead of VehicleDataStatus type
+-- trailerSwayControl is updated to string instead of VehicleDataStatus type
+local hmiResParams_invalidBothParams = {
+  stabilityControlsStatus = {
+    escSystem = 30.2,
+    trailerSwayControl = "invalid"
+  },
+  speed = 30.2
+}
+
+-- stabilityControlsStatus in not a structure
+local hmiResParams_stabilityControlsStatusIsNotStructure = {
+  stabilityControlsStatus = "ON",
+  speed = 30.2
+}
+
+-- stabilityControlsStatus parameter absent
+local hmiResParams_stabilityControlsStatusAbsent = {
```

Comment:
@ydementieiev As a suggestion - you can put all these structures in one table and go through it in a loop, as was done here https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/16/files#diff-82ed425198b71780df166654274cbccb

Reply:
@LitvinenkoIra thank you , if it is only suggestion, I would like to leave an option where each case has its own message. If something will wrong with this case we can see clearly message in console about what was wrong without opening the code